### PR TITLE
Stopgap: MongoDB $jsonSchema

### PR DESCRIPTION
**Pull request by Druue**
part of https://github.com/prisma/schema-team/issues/403

### CODE REVIEW

File: schema-engine/connectors/mongodb-schema-connector/src/sampler/statistics.rs [hunk 1]
```diff
@@ -147,6 +148,12 @@ impl<'a> Statistics<'a> {
                     None => renderer::datamodel::Model::new(name),
                 };
 
+                if let Some(walker) = doc_count.collection_walker {
+                    if walker.has_schema() {
+                        model.documentation("json schema msg")
+                    }
+                }
+
```
Comment (Druue) [inline schema-engine/connectors/mongodb-schema-connector/src/sampler/statistics.rs:156]: This adds the documentation message that goes into the schema file above the model

File: schema-engine/connectors/schema-connector/src/warnings.rs [hunk 2]
```diff
@@ -408,6 +410,12 @@ impl fmt::Display for Warnings {
             f,
         )?;
 
+        render_warnings(
+            "The following models have a JSON Schema defined in the database, which is not yet fully supported. Read more: https://pris.ly/d/todo",
+            &self.json_schema_defined,
+            f
+        )?;
```
Comment (Druue) [inline schema-engine/connectors/schema-connector/src/warnings.rs:417]: This adds the cli warning message

Comment (jkomyno) [reply schema-engine/connectors/schema-connector/src/warnings.rs:417]: Also, let's make sure we give the `pris.ly` link a proper URL. https://pris.ly/d/mongodb-jsonschema should be good enough

**Review by tomhoule — COMMENTED** [review]
The logic looks good :+1: I think we only need the messages work, indeed.

**Review by jkomyno — APPROVED** [review]

**Review by janpio — APPROVED** [review]

**Pull request merged**